src/arch/xen/xenstore.c

```diff
@@ -0,0 +1,221 @@
+#include "xenstore.h"
```

Comment:
There is no copyright here. Have the source any? If so, please add it (and yours too)